### DISCUSSION TITLE
希望能增加一个，不用访问密码和key就能使用的10条免费限制，然后输入key或者访问密码

### DISCUSSION THREAD

**Discussion by Dashan-IZ** [ChatGPTNextWeb/NextChat#1038] [Ideas]
希望能增加一个，不用访问密码和key就能使用的10条免费限制，然后输入key或者访问密码，免费的10条从.env的key走

**Comment by mask71** [comment]
+1

**Comment by chengkeminsu** [comment]
+1

**Comment by Yidadaa** [comment]
我也希望能增加此功能，期待你的 pr 贡献代码。

**Reply by gtoxlili** [reply]
理论上要真做这个功能就得要有数据库了吧。如果是以localStorage存在前端的话安全问题有点大的

**Reply by OldDream666** [reply]
是的，没有检测机制，本地的话清除一下我又可以用了

**Reply by Dashan-IZ** [reply]
用缓存也行的，浏览器有很多密码缓存，很多人不愿意清楚缓存的，虽然清除后可以刷新10天，大多数人也不会
